generate-community/src/bin/validate.rs

```diff
@@ -53,7 +54,7 @@ fn validate_node(node: &CommunityNode) -> Result<(), String> {
             };
 
             if let Some(bio) = &member.bio {
-                if bio.chars().count() > MAX_BIO_LENGTH {
+                if bio.graphemes(true).count() > MAX_BIO_LENGTH {
                     Err(format!("Bio is longer than the maximum allowed length of {}. It is currently {} characters long.", MAX_BIO_LENGTH, bio.len()))?;
```

Comment:
The error reporting still reports in byte length

Reply:
:facepalm: